app/serializers/api/participant_serializer.rb

```diff
@@ -0,0 +1,50 @@
+module API
+  class ParticipantSerializer < Blueprinter::Base
+    class AttributesSerializer < Blueprinter::Base
```

Comment:
I liked the [view structure we used in NPQ reg](https://github.com/DFE-Digital/npq-registration/blob/main/app/serializers/api/participant_serializer.rb#L9), I think we should use it again for the API versions here